source/extensions/quic_listeners/quiche/platform/envoy_quic_clock.cc

```diff
@@ -0,0 +1,22 @@
+#include "extensions/quic_listeners/quiche/platform/envoy_quic_clock.h"
+
+namespace Envoy {
+namespace Quic {
+
+quic::QuicTime EnvoyQuicClock::ApproximateNow() const {
+  // This might be expensive as Dispatcher doesn't store approximate time_point.
+  return Now();
+}
+
+quic::QuicTime EnvoyQuicClock::Now() const {
+  return quic::QuicTime::Zero() + quic::QuicTime::Delta::FromMicroseconds(
+                                      microsecondsSinceEpoch(time_system_.monotonicTime()));
```

Comment:
nit: remove empty line.

Reply:
which line?

Reply:
Sorry, it looked like there is an empty line between line 12 and 13, but it's not.